src/main/java/com/instamojo/wrapper/util/HttpUtils.java

```diff
@@ -31,10 +28,34 @@
 
     private static final Logger LOGGER = Logger.getLogger(HttpUtils.class.getName());
 
+    private static String userAgent = "";
+
+    static {
+        String osInfo = System.getProperty("os.name") + "/" + System.getProperty("os.version");
+        String langInfo = "java/" + System.getProperty("java.version");
+
+        String libraryVersion = "lib-ver-not-found";
```

Comment:
We can leave out the library version if this information is not available or (IMO) it's better to use the `unknown`.